### ISSUE TITLE
No .NET support

### ISSUE THREAD

**Issue by LarsKemmann** [payeezy/payeezy_direct_API#28]
A .NET Standard (1.1/2.0) library would be nice to have. Thanks!

**Comment by nhustak** [comment]
Second this.

**Comment by LarsKemmann** [comment]
@nhustak https://developer.payeezy.com/payeezy-api/apis/

**Comment by nhustak** [comment]
@LarsKemmann  Yeah thanks!!!  I had realized right after I posted this they did it one better and  with full on functional examples.

**Comment by WillStrohl** [comment]
+1 for .NET library and .NET Core code samples.

**Comment by morenoh149** [comment]
maybe if they provided detailed curl requests and HTTP requests (the full http request as seen over the wire) .net folks and any other stack could make their own solutions (and push it to this repo).

update: spoke too soon there are curl examples here you can reimplement for .net https://github.com/payeezy/payeezy_direct_API/tree/master/payeezy_curl
